workspaces/ui/src/components/diff/review-diff/PathAndMethod.js

```diff
@@ -116,21 +121,33 @@ export const PathAndMethodOverflowFriendly = ({ path, method }) => {
     </Typography>
   );
 
-  return (
-    <LightTooltip title={<PathAndMethod method={method} path={path} />}>
+  if (path.length > 47) {
```

Comment:
What is the problem & solution here?

Reply:
extra long paths in the diff review page. now you can hover to reveal the full path

Reply:
Where does 47 come from? Is that portable? The component should handle overflow anyway and it seems more consistent to put the tooltip on every row